src/vscode-bicep/src/deploy/tree/registerTrees.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { AzExtTreeDataProvider } from "@microsoft/vscode-azext-utils";
+
+import { ext } from "../../extensionVariables";
+import { AzLocationTreeItem } from "./AzLocationTreeItem";
+import { AzManagementGroupTreeItem } from "./AzManagementGroupTreeItem";
+import { AzResourceGroupTreeItem } from "./AzResourceGroupTreeItem";
+
+export function registerTrees(): void {
+  const azLocationTreeItem: AzLocationTreeItem = new AzLocationTreeItem();
```

Comment:
`AzLocationTreeItem` and `AzResourceGroupTreeItem` below extends `AzureAccountTreeItemBase` which implements `Disposable`. They should be registered by using the `Disposable` helper class defined in `utils/disposable.ts`.

Reply:
Ah thanks @shenglol. Updated here:
https://github.com/Azure/bicep/commit/a346eb2ab6385b9f896d513c345116de6bb425e6